internal/services/network/express_route_connection_resource.go

```diff
@@ -197,10 +192,17 @@ func resourceExpressRouteConnectionCreate(d *pluginsdk.ResourceData, meta interf
 			RoutingConfiguration:      expandExpressRouteConnectionRouting(d.Get("routing").([]interface{})),
 			RoutingWeight:             pointer.To(int64(d.Get("routing_weight").(int))),
 			ExpressRouteGatewayBypass: pointer.To(d.Get("express_route_gateway_bypass_enabled").(bool)),
-			EnablePrivateLinkFastPath: pointer.To(d.Get("private_link_fast_path_enabled").(bool)),
 		},
 	}
 
+	privateLinkFatPath := d.GetRawConfig().AsValueMap()["private_link_fast_path_enabled"]
```

Comment:
Could we be mindful of the spelling here 
```suggestion
	privateLinkFastPath := d.GetRawConfig().AsValueMap()["private_link_fast_path_enabled"]
```

Reply:
fixed